requirements/requirements.in

```diff
@@ -4,7 +4,7 @@
 #
 cryptography
 Django>=4.2.5,<4.3.0
-djangorestframework
+djangorestframework==3.14.0
```

Comment:
A `<=` pin might be okay, I'm worried that such an opinionated pin here might cause downstream packaging issues. Even if a service had a newer DRF, it only causes issues with the authenticator app as far as we know.